sourcemash/frontend/static/js/default.js

```diff
@@ -14,4 +16,12 @@ $(document).ajaxError(function (e, xhr, options) {
         fieldErrors.text(errors[errorField]);
     }
   }
-});
\ No newline at end of file
+
+  // Show Modal, if Not Authenticated
+  var errorStatus = xhr.status;
+  if (errorStatus == 401) {
```

Comment:
The variable name makes the code a bit clearer, but I'm not emotionally committed if you want it gone.